Error on testing development copy

<!-- This comments are hidden when you submit the issue,
so you do not need to remove them! -->

<!-- Please be sure to check out our contributing guidelines,
https://github.com/astropy/astropy/blob/master/CONTRIBUTING.md .
Please be sure to check out our code of conduct,
https://github.com/astropy/astropy/blob/master/CODE_OF_CONDUCT.md . -->

<!-- Please have a search on our GitHub repository to see if a similar
issue has already been posted.
If a similar issue is closed, have a quick look to see if you are satisfied
by the resolution.
If not please go ahead and open an issue! -->

<!-- Please check that the development version still produces the same bug.
You can install development version with
pip install git+https://github.com/astropy/astropy
command. -->

### Description
<!-- Provide a general description of the bug. -->
Getting an error on testing development copy of astropy.

### Expected behavior
<!-- What did you expect to happen. -->
There should be no errors or failures.

### Actual behavior
<!-- What actually happened. -->
<!-- Was the output confusing or poorly described? -->
I'm getting following error.
```python
ImportError while importing test module '/home/param/opensource/astropy/astropy/coordinates/tests/test_name_resolve.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../miniconda3/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
astropy/astropy/coordinates/tests/test_name_resolve.py:21: in <module>
    from pytest_remotedata.disable_internet import no_internet
E   ModuleNotFoundError: No module named 'pytest_remotedata'
```

### Steps to Reproduce
<!-- Ideally a code example could be provided so we can run it ourselves. -->
<!-- If you are pasting code, use triple backticks (```) around
your code snippet. -->
<!-- If necessary, sanitize your screen output to be pasted so you do not
reveal secrets like tokens and passwords. -->

1. Clone the repository 
2. `>> cd astropy`
3. `>> pip install -e .`
4. `>> cd ..`
5. `>> pytest`



### System Details
<!-- Even if you do not think this is necessary, it is useful information for the maintainers.
Please run the following snippet and paste the output below:
import platform; print(platform.platform())
import sys; print("Python", sys.version)
import numpy; print("Numpy", numpy.__version__)
import astropy; print("astropy", astropy.__version__)
import scipy; print("Scipy", scipy.__version__)
import matplotlib; print("Matplotlib", matplotlib.__version__)
-->
**platform :** 'Linux-4.19.128-microsoft-standard-x86_64-with-glibc2.10'
**sys :**  '3.8.3 (default, May 19 2020, 18:47:26) \n[GCC 7.3.0]'
**numpy :** '1.19.2'
**astropy :** '4.2.dev1241+g883723e05'

--- a/docs/development/workflow/get_devel_version.rst
+++ b/docs/development/workflow/get_devel_version.rst
@@ -274,6 +274,9 @@ Testing is an important part of making sure astropy produces reliable,
 reproducible results. Before you try out a new feature or think you have found
 a bug make sure the tests run properly on your system.
 
+before runnig tests, see
+:ref:`testing-dependencies`.
+
 If the test *don't* complete successfully, that is itself a bug--please
 `report it <https://github.com/astropy/astropy/issues>`_.
 

--- a/docs/development/workflow/get_devel_version.rst
+++ b/docs/development/workflow/get_devel_version.rst
@@ -274,8 +274,7 @@ Testing is an important part of making sure astropy produces reliable,
 reproducible results. Before you try out a new feature or think you have found
 a bug make sure the tests run properly on your system.
 
-before runnig tests, see
-:ref:`testing-dependencies`.
+Before running your tests, please see :ref:`testing-dependencies`.
 
 If the test *don't* complete successfully, that is itself a bug--please
 `report it <https://github.com/astropy/astropy/issues>`_.

